docs/csharp/language-reference/keywords/implicit-numeric-conversions-table.md

```diff
@@ -1,47 +1,50 @@
-|[short](../../../csharp/language-reference/keywords/short.md)|`int`, `long`, `float`, `double`, or `decimal`|  
-|[ushort](../../../csharp/language-reference/keywords/ushort.md)|`int`, `uint`, `long`, `ulong`, `float`, `double`, or `decimal`|  
-|[int](../../../csharp/language-reference/keywords/int.md)|`long`, `float`, `double`, or `decimal`|  
-|[uint](../../../csharp/language-reference/keywords/uint.md)|`long`, `ulong`, `float`, `double`, or `decimal`|  
-|[long](../../../csharp/language-reference/keywords/long.md)|`float`, `double`, or `decimal`|  
-|[char](../../../csharp/language-reference/keywords/char.md)|`ushort`, `int`, `uint`, `long`, `ulong`, `float`, `double`, or `decimal`|  
-|[float](../../../csharp/language-reference/keywords/float.md)|`double`|  
-|[ulong](../../../csharp/language-reference/keywords/ulong.md)|`float`, `double`, or `decimal`|  
+|[sbyte](sbyte.md)|`short`, `int`, `long`, `float`, `double`, or `decimal`|  
+|[byte](byte.md)|`short`, `ushort`, `int`, `uint`, `long`, `ulong`, `float`, `double`, or `decimal`|  
+|[short](short.md)|`int`, `long`, `float`, `double`, or `decimal`|  
+|[ushort](ushort.md)|`int`, `uint`, `long`, `ulong`, `float`, `double`, or `decimal`|  
+|[int](int.md)|`long`, `float`, `double`, or `decimal`|  
+|[uint](uint.md)|`long`, `ulong`, `float`, `double`, or `decimal`|  
+|[long](long.md)|`float`, `double`, or `decimal`|  
+|[char](char.md)|`ushort`, `int`, `uint`, `long`, `ulong`, `float`, `double`, or `decimal`|  
+|[float](float.md)|`double`|  
+|[ulong](ulong.md)|`float`, `double`, or `decimal`|  
   
 ## Remarks  
+
+- Any [integral type](integral-types-table.md) is implicitly convertible to any [floating-point type](floating-point-types-table.md).
+
+- Precision but not magnitude might be lost in the conversions from `int`, `uint`, `long`, or `ulong` to `float` and from `long` or `ulong` to `double`.  
   
--   Precision but not magnitude might be lost in the conversions from `int`, `uint`,  `long`, or `ulong` to `float` and from `long` or `ulong` to `double`.  
-  
--   There are no implicit conversions to the `char` type.  
-  
--   There are no implicit conversions between floating-point types and the `decimal` type.  
+- There are no implicit conversions to the `char` type.  
   
--   A constant expression of type `int` can be converted to `sbyte`, `byte`, `short`, `ushort`, `uint`, or `ulong`, provided the value of the constant expression is within the range of the destination type.  
+- There are no implicit conversions between the `float` and `double` types and the `decimal` type.  
   
-## C# Language Specification  
- [!INCLUDE[CSharplangspec](~/includes/csharplangspec-md.md)]  
+- A constant expression of type `int` can be converted to `sbyte`, `byte`, `short`, `ushort`, `uint`, or `ulong`, provided the value of the constant expression is within the range of the destination type.  
```

Comment:
This is mystifying -- it's not clear why this paragraph is here, though it should be. An implicit conversion of an out-of-range constant generates a compiler error (CS0031) regardless of context.